lib/Backends/OpenCL/OpenCL.h

```diff
@@ -16,8 +16,6 @@
 #ifndef GLOW_BACKENDS_OPENCL_OPENCL_H
 #define GLOW_BACKENDS_OPENCL_OPENCL_H
 
-#include "OpenCLDeviceManager.h"
```

Comment:
This change in "include" direction actually makes a lot of sense to me. Just wanted to note that CPU / Habana backends header files include the CPU / Habana device manager header (correspondingly, both cases only because of createDeviceManager(...) implemented in the header). This seems like the correct direction.

Reply:
Yeah we should just fix this everywhere I think. (not in this PR!)